src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -260,6 +263,20 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
       },
     },
   },
+  Loft: {
+    description: 'Create a 3D body by blending between two or more sketches',
+    icon: 'loft',
+    needsReview: true,
+    args: {
+      selection: {
+        inputType: 'selection',
+        selectionTypes: ['solid2D'],
+        multiple: true,
```

Comment:
This shows the message _Please select one or more faces_ but we actually need _two or more_ in the case of lofts.

Reply:
Ah you raise a great point about this selection argument needing a better way to specify its validation. We've brushed across this problem before with other flows but we should consider making this work properly now (or as a follow-up if this flow feels good enough to ship).

What if we alter `selection` type arguments to take a `validation` config, which is just a function which takes the current `Selections` and returns a `true | string`. Then command authors can set their custom validation logic in there, for example saying that the selection should, for example, contain 2 or more items within `graphSelections`, and return the message if they don't. Then in the `CommandBarSelectionInput` component we surface the validation message and disallow submission if a `string` is returned.

Reply:
This is a great proposal. Created https://github.com/KittyCAD/modeling-app/issues/4661 to address is in a dedicated PR. TY!